packages/hoppscotch-common/package.json

```diff
@@ -76,6 +76,7 @@
     "postman-collection": "^4.2.0",
     "process": "^0.11.10",
     "qs": "^6.11.2",
+    "quicktype-core": "^23.0.79",
```

Comment:
Since you have added a dependency, you would have had an updated `pnpm-lock.yaml` file. Please commit that in as well since we require lockfiles to be up-to-date with dependencies.

Reply:
Sure thing!

Reply:
@AndrewBastin done. Can you review now?